src/main/java/io/confluent/examples/streams/PageViewRegionExampleDriver.java

```diff
@@ -108,13 +106,12 @@ private static void consumeOutput(String bootstrapServers) {
     consumerProperties.put(ConsumerConfig.GROUP_ID_CONFIG,
       "pageview-region-lambda-example-consumer");
     consumerProperties.put(ConsumerConfig.AUTO_OFFSET_RESET_CONFIG, "earliest");
-    final KafkaConsumer<String, Long> consumer = new KafkaConsumer<>(consumerProperties);
 
-    consumer.subscribe(Collections.singleton(resultTopic));
-    while (true) {
-      final ConsumerRecords<String, Long> consumerRecords = consumer.poll(Long.MAX_VALUE);
-      for (final ConsumerRecord<String, Long> consumerRecord : consumerRecords) {
-        System.out.println(consumerRecord.key() + ":" + consumerRecord.value());
+    try (final KafkaConsumer<String, Long> consumer = new KafkaConsumer<>(consumerProperties)) {
+      consumer.subscribe(Collections.singleton(resultTopic));
+      while (true) {
+        final ConsumerRecords<String, Long> consumerRecords = consumer.poll(Long.MAX_VALUE);
+        consumerRecords.forEach(record -> System.out.println(record.key() + ":" + record.value()));
```

Comment:
We don't want to use Lambdas to be Java7 compliant. Cf. JavaDoc PageViewRegionExample.java#L43

\cc @miguno Or is it ok in the driver to change it?

Reply:
@mjsax 
In the same example, `IntStream` is used which is Java8, so I have changed this one. No issues I'm happy to revert it back.

Reply:
@miguno Should the example be Java7 compliant or not?

I personally don't have a strong opinion. If it is Java8 already, it fine to update. Otherwise we need to fix `IntStream`, too... Let's wait for @miguno reply.

Reply:
I'd stick to Java 7 here.  There's an equivalent example `PageViewRegionLambdaExample` where we area using Java 8+.

Reply:
I have updated the `driver` class to use Java 7. 

Note that, many existing driver classes are still using `IntStream` which is introduced in Java 8. If ok to refactor, I'm ready to open a PR for them too.